guides/hiring/remote_test/csharp_marking_guide.md

```diff
@@ -0,0 +1,28 @@
+# Marking guide
+
+60 marks in total. There are no half marks. 80% (48 marks) required to pass.
```

Comment:
> We pretty much recommend static code analysis for....

Fair.

> To see if a candidate can Install and set up a command line tool

God I hope so.

> This isn't gherkin syntax. Capybara provides browser drivers

My mistake, I see this is the ruby interface for selenium I quickly skimmed the capybara readme and misread. Pretty much all languages have a library for selenium so agreed.